components/k8s-dashboard-proxy/README.md

```diff
@@ -0,0 +1,32 @@
+# Kubernetes Dashboard proxy
+
+## Overview
+
+The Kubernetes Dashboard proxy is a transparent proxy for the Kubernetes Dashboard allowing the Kyma clients to control the access to the Kubernetes Dashboard. Even though by default you don't need any verification to access the the Dashboard, the proxy uses Dex identity provider to ensure the Dashboard users are authenticated and authorized.
+
+## Concept
+
+The reverse proxy is a very thin layer Go application used by the Kubernetes Dashboard. It is used as an upstream for the [Keycloak Proxy](https://github.com/keycloak/keycloak-gatekeeper) which acts as a gateway for the Kubernetes Dashboard proxy. 
+
+The Keycloak proxy forwards the requests to Dex identity service, asking the user for authentication credentials. Based on the authentication credentials correctness, Dex checks if you are authorized to access the Kubernetes Dashboard.
```

Comment:
`The Keycloak proxy forwards the requests to Dex identity service, and asks you for authentication credentials.`